util/db.py

```diff
@@ -41,6 +40,18 @@ def get_account_by_email(cls, email: str) -> dict:
         if account_data is not None:
             account_data.pop('_id')
         return account_data
+    
+    @classmethod
+    def get_account_by_authorization(cls, key: str, value: str) -> list[dict]:
+        """
+        Finds an account from the mongo database given an email addres.
+
+        :param email: The email address of the account.
```

Comment:
The method doesn't take an 'email' param. Should the email address be the 'value' param?